frontend/components/organisms/AnswerComponent/index.tsx

```diff
@@ -31,19 +31,17 @@ const AnswerComponent = () => {
         reValidateMode: 'onSubmit',
     })
 
-    useEffect(() => {
-    }, [isDisableSubmit])
+    useEffect(() => {}, [isDisableSubmit])
```

Comment:
Is this used ? please delete if it does nothing

Reply:
This is not part of the changes for this PR